### ISSUE TITLE
Localize content

### ISSUE THREAD

**Issue by sjsyrek** [sjsyrek/haskell-study-startup#4]
I would love to have this repo translated into multiple languages. Any volunteers?

**Comment by sjsyrek** [comment]
Full localization would require both the original README.md and the guidelines.md file to be translated. Thanks for your help!

**Comment by LW001** [comment]
I could do german if you still need it

**Comment by sjsyrek** [comment]
Yes, German is at the top of my list... because I live in Germany! Thank you.

**Comment by mgh-ding** [comment]
I'd like to help translate it into Chinese, if it's still needed :)
